src/main/java/com/sprint/mission/discodeit/controller/BinaryContentController.java

```diff
@@ -0,0 +1,39 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+import java.util.UUID;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping(value = "/api/binaryContent")
```

Comment:
url은 보통 카멜케이스보단 `-`(하이픈)을 사용해요.  
```suggestion
@RequestMapping(value = "/api/binary-content")
```